src/opm/io/eclipse/rst/network.cpp

```diff
@@ -193,6 +193,7 @@ namespace {
                 usys.to_si(Opm::UnitSystem::measure::pressure,
                            rnode[VI::RNode::index::PressureLimit]);
         }
+        node.pressure = rnode[VI::RNode::index::NodePres];
```

Comment:
Might you need unit conversion here?  The `rnode` values are **typically** in output units so depending on what code uses the `Node::pressure` value there could be an inconsistency here.

Reply:
I believe you are right, will fix. Thanks!